testsuite/tests/kuadrant/authorino/wristband/test_wristband.py

```diff
@@ -0,0 +1,27 @@
+"""Edge Authentication tests"""
+from jose import jwt
+
+
+def test_wristband_token_claims(oidc_provider, auth, wristband_token, wristband_endpoint, certificates):
+    """Verify acquired jwt token claims"""
+    wristband_decoded = jwt.decode(wristband_token, certificates["signing_ca"].certificate)
+    assert wristband_decoded["exp"] - wristband_decoded["iat"] == 300
+    assert wristband_decoded["iss"] == wristband_endpoint
+    # check differences in claims between rhsso token and acquired wristband token
+    access_token_decoded = jwt.decode(auth.token.access_token, oidc_provider.get_public_key(), audience="account")
```

Comment:
There is a `decode_token` method in the python-keycloak library

Reply:
This method uses a hard-coded audience claim check as client id, which does not match the actual audience value (it is a known issue https://stackoverflow.com/questions/53550321/keycloak-gatekeeper-aud-claim-and-client-id-do-not-match). There is an option to not verify audience and use this build-in method, but I would prefer to decode it by using the jwt library (as it is now).